example/src/ModifyQuantity/ModifyQuantityPage.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import { ModifyQuantity, Screen, useTheme } from 'smartway-react-native-ui';
+
+export const ModifyQuantityPage = () => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        container: {
+            alignItems: 'center',
+            marginTop: theme.sw.spacing.m,
+        },
+    });
+
+    const [value, setValue] = useState<number>(0);
+
+    const onAdd = () => {
```

Comment:
I think the logic when we increment or decrement the value should be kept within the ModifyQuantity component, the caller only needs to know when the value has changed.

Therefore, I would rather see a `onValueChange` property that takes a callback with the changed value in parameters ``<ModifyQuantity onChange={(value) => setValue(value)}` />

Reply:
updated!